### ISSUE TITLE
Tree views are impractical in mobile view

### ISSUE THREAD

**Issue by gplanchat** [orocommerce/orocommerce#4]
Tree views (Categories, CMS pages, etc) does not display in a practical manner on tablet view and are unusable on smartphones.

Maybe some alternative display could be considered, such as a sliding tree menu such as eg a [sliding menu](http://codecanyon.net/item/jquery-infinite-sliding-menu-plugin/full_screen_preview/164115).

**Comment by mbessolov** [comment]
I agree that occasionally a user might need to make a quick change (fix a typo, etc.) while en route and not having access to a device with a larger resolution. I would not expect the users though to perform content management on a tablet device or smartphone. 

We are open for suggestions and community contributions - a UI that provides good usability on touch screen / small screen devices and allows to perform majority of the content management functions (effective tree navigation, moving the nodes in the tree with a drag-n-drop alternative) could be a good solution here.

**Comment by maurosergiosilva** [comment]
Mybe https://tympanus.net/Development/MultiLevelPushMenu/ could be used.

**Comment by mbessolov** [comment]
Thank you for the suggestion, we'll check it.

**Comment by maurosergiosilva** [comment]
These seams interesting too: http://adnantopal.github.io/slimmenu/
